apps/condo/domains/ticket/components/TicketHint/TicketIdHintCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { InfoCircleOutlined } from '@ant-design/icons'
+import styled from '@emotion/styled'
+import { Alert, Typography } from 'antd'
+import { get } from 'lodash'
+import Link from 'next/link'
+
+import { colors, fontSizes } from '@condo/domains/common/constants/style'
+import { useIntl } from '@core/next/intl'
+import { CSSProperties, useMemo } from 'react'
+
+import { TicketHint, TicketHintProperty } from '../../utils/clientSchema'
+import { TicketHintContent } from './TicketHintContent'
+
+const StyledAlert = styled(Alert)`
+  background-color: ${colors.successBG};
+  border: none;
+  
+  & svg, & > .ant-alert-content > .ant-alert-message {
+   color: ${colors.black}; 
+  }
+`
+
+const TEXT_STYLES: CSSProperties = { color: colors.black }
+const LINK_STYLES: CSSProperties = { ...TEXT_STYLES, position: 'relative', bottom: '-7px', fontSize: fontSizes.content }
+const TICKET_HINT_CONTENT_STYLES: CSSProperties = { maxHeight: '5em' }
+
+export const TicketIdHintCard = ({ propertyId }) => {
```

Comment:
Name overlaps with `Card` component from Ant.
`TicketId…` confuses in terms of displaying id of a ticket.